### PR TITLE
Chk 1252 api review

### PR DESCRIPTION
**Pull request by pietro-tota**
Depends on pagopa/pagopa-infra#794
#### List of Changes

- Open api review: add missing information such as contact, DR link, fields descriptions etc
- Removed unused activateTransaction method (that return 501 since not anymore used because of the usage of the new Nodo api's)

#### Motivation and Context
Review open api specifications

#### How Has This Been Tested?
Junit tests

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: api-spec/transactions-api.yaml [hunk 1]
```diff
@@ -111,44 +131,48 @@ paths:
...
                 $ref: "#/components/schemas/ProblemJson"
-        500:
+        '500':
           description: Internal server error
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ProblemJson"
-        504:
+        '504':
```
Comment (infantesimone) [inline api-spec/transactions-api.yaml:164]: 504 is returned by the app gtw, so it's ok only for spec on infra. what do i think?

Comment (pietro-tota) [reply api-spec/transactions-api.yaml:164]: Yes, I agree, my bad

**Review by infantesimone — DISMISSED** [review]
just a comment, but sounds good :-)

File: api-spec/transactions-api.yaml (hunk 1 above)
Comment (pietro-tota) [reply api-spec/transactions-api.yaml]: Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/165/commits/a48968b5c0161b8a10df153d3c594a16dced3c5c

**Review by infantesimone — APPROVED** [review]

**Pull request merged**